cmd/internal/find/primary/meta.go

```diff
@@ -172,6 +172,9 @@ matches 'foo'.
       Returns true if some element in m['foo'] is an object o s.t.
       p(o['bar']) returns true.
 
+NOTE: You can use a backslash "\" to escape ".", "[", or "]". For example,
+'.foo\.bar p' returns p(m['foo.bar']).
```

Comment:
Where does the `m` come from in this key sequence?

Reply:
The examples above the `NOTE` use `m` to reference the entry's metadata, so I think people shouldn't have trouble figuring that out.